src/sprite.c

```diff
@@ -363,31 +355,35 @@ s32 func_802DE894(s32 arg0, s32 arg1, s32 arg2, s32 arg3, s32 arg4, s32 arg5, s3
 
 INCLUDE_ASM(s32, "sprite", func_802DE8DC);
 
-#ifdef NON_EQUIVALENT
+typedef struct unkStructSprite {
```

Comment:
UnkStructSprite - also I think UnkSpriteStruct is better, cause the other way sounds like this is a sprite and not a struct